docs/source/getting-started.rst

```diff
@@ -81,25 +86,28 @@ In this section, we will create a new notebook called
         interactive.panel()
     ).servable()
 
+ You can exit the running Jupyter Notebook program using CTRL+C in your terminal or command line.
 
 .. _hvPlot interactive: https://raw.githubusercontent.com/holoviz/hvplot/master/examples/user_guide/Interactive.ipynb
 .. _quickstart: https://raw.githubusercontent.com/Anaconda-Platform/anaconda-project/master/examples/quickstart/Interactive.ipynb
 
 Run your project
 ================
 
-#. Register a new command to launch the notebook as a `Panel`_ app::
+1. Register a new command to launch the notebook as a `Panel`_ app::
 
      $ anaconda-project add-command --type unix dashboard "panel serve Interactive.ipynb"
 
-.. note:: The ``unix`` command type may be used for linux & macOS. For Windows, replace ``--type unix`` with ``--type win``
-.. _Panel: https://panel.holoviz.org
+  .. note:: The ``unix`` command type may be used for linux & macOS. For Windows, replace ``--type unix`` with ``--type win``
```

Comment:
Can you change `--type win` to `--type windows`?

Reply:
Sure thing